sdk/healthinsights/health-insights-radiologyinsights-rest/README.md

```diff
@@ -7,14 +7,7 @@
 **Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
 
 Key links:
-
-Key links:
-
-- [Source code]
-- [Package (NPM)]
-- [API reference documentation]
-- [Product Information](https://docs.microsoft.com/rest/api/maps/route)
-- [Samples]
+[Source code] | [Package (NPM)] | [API reference documentation] | [Product Information] | [Samples]
```

Comment:
Product Information should point to: https://learn.microsoft.com/en-us/azure/azure-health-insights/radiology-insights/
Currently pointing to java